.schema/config.schema.json

```diff
@@ -260,6 +260,13 @@
           ],
           "default": 302
         },
+        "url_param": {
```

Comment:
Let's rename this to `return_to_query_param` to make it more obvious :)

Reply:
@aeneasr done